File: Curso_logica_de_programacao/Exercicios_logica_em_C++/Retangulo.c/Retangulo.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <math.h>

using namespace std;

int main(){

        double base, altura, comprimento, perimetro, diagonal, area;

        cout << "Base do retangulo: ";
        cin >> base;
        cout << "Altura do retangulo: ";
        cin >> altura;

        area = base * altura;
        perimetro = (2 * base) + (2 * altura);
        diagonal = sqrt(pow(base,2) + pow(altura,2));

        cout << fixed << setprecision(4);
        cout << "\nArea = " << area;
        cout << "\nPerimetro = "<< perimetro;
        cout << "\nDiagonal = " << diagonal;

    return 0;
}

File: Algoritmos_e_Logica_de_Programacao/Exercicios_logica_em_Java/Retangulo/src/retangulo.java
import java.util.Scanner;
import java.util.Locale;

public class retangulo {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.print("Base do retangulo: ");
        double base = sc.nextDouble();
        System.out.print("Altura do retangulo: ");
        double altura = sc.nextDouble();
        
        double area = base * altura;
        double perimetro = 2 * (base + altura);
        double diagonal = Math.sqrt(Math.pow(base,2) + Math.pow(altura,2));

        System.out.println("Area = " + String.format("%.2f", area));
        System.out.println("Area = " + String.format("%.2f", perimetro));
        System.out.println("Area = " + String.format("%.2f", diagonal));

        sc.close();
    }
}
